ci/build_wheel.sh

```diff
@@ -26,6 +26,14 @@ SKBUILD_CMAKE_ARGS="-DDETECT_CONDA_ENV=OFF;-DDISABLE_DEPRECATION_WARNINGS=ON;-DC
     --disable-pip-version-check
 
 mkdir -p final_dist
-python -m auditwheel repair -w final_dist dist/*
+python -m auditwheel repair -w final_dist \
+  --exclude "libcublas.so.12" \
```

Comment:
Can we use exclude patterns like `libcublas.so.*`? How does this work for CUDA 11 / support multiple future major versions? Should we skip the excludes for CUDA 11?

Reply:
I didn't look thoroughly into whether auditwheel supports wildcards. The documentation didn't say anything about it, but I didn't delve into the source code to find out. That'll be my next step.

Reply:
It seems that wildcards are not supported:

https://github.com/pypa/auditwheel/blob/7c3501bbd5434bdc82f6b24effa2de947f1d778f/src/auditwheel/lddtree.py#L408

Think this would be worth contributing to auditwheel?

Reply:
I’d support a feature to auditwheel for this.

Reply:
I've made an auditwheel branch for this. I'll upstream it once we get approval.

Reply:
I opened a PR to auditwheel:

https://github.com/pypa/auditwheel/pull/508

However, it looks like someone else tried something similar, and upstream didn't seem very interested:

https://github.com/pypa/auditwheel/pull/411

Reply:
I've added a `case` statement to decide which `--exclude` arguments to pass based on the CUDA version. This will allow us to `--exclude` the correct libraries even without wildcard support from upstream.

Reply:
Thanks -- I am fine with this approach.